common/lib/capa/capa/inputtypes.py

```diff
@@ -310,6 +311,8 @@ def _get_render_context(self):
             (a, v) for (a, v) in self.loaded_attributes.iteritems() if a in self.to_render
         )
         context.update(self._extra_context())
+        if self.answervariable:
+            context.update({'answervariable': self.answervariable})
```

Comment:
We need ```answervariable``` in ```optioninput``` problem only and lots of other problem's tests were failing due to this that's why I'm adding it into context when its not ```None```.